examples/sdp_examples/Converting a non-dcp compliant problem to Semi-Definite Programming Problem.ipynb

```diff
@@ -0,0 +1,111 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#  Non-DCP Compliant Problem to Semi-Definite Programming Problem \n",
+    "\n",
```

Comment:
I'd suggest adding a bit of intro here: what's the idea of what you're doing below? For what kinds of problems should a newbie expect it to work?